9.secretstore/README.md

```diff
@@ -0,0 +1,154 @@
+# Secrets API
+
+This tutorial will help you write a node app to access secrets stored in a Secret Store using the Dapr Secrets API. For this sample, we will be using Kubernetes secret store
```

Comment:
Can we change this overview to.
"This tutorial shows you how to use the Dapr secrets API to access secrets from secret stores. For sample uses a Node application to access a Kubernetes secret store"

Reply:
fixed